flixel/input/gamepad/FlxGamepad.hx

```diff
@@ -486,4 +630,14 @@ enum FlxAxes
 {
 	X;
 	Y;
+}
+
+enum GamepadModel
+{
+	Logitech;
+	OUYA;
+	PS3;
+	PS4;
+	Xbox;
```

Comment:
It probably should be. There's an XBox ONE controller too which we shouldn't assume is the same. I was just following what we had already, which just had "XBoxInput" -- I'll change all xbox classes to XBox360